monkey/infection_monkey/master/automated_master.py

```diff
@@ -15,6 +15,7 @@
 from infection_monkey.utils.threading import create_daemon_thread, interruptible_iter
 from infection_monkey.utils.timer import Timer
 
+from ..post_breach.custom_pba.custom_pba import CustomPBA
```

Comment:
Add `CustomPBA` in `__init__` and use it from there. Just for readability and consistency.